jaxrs-tck/src/main/java/jakarta/ws/rs/tck/ee/rs/ext/providers/JAXRSProvidersClientIT.java

```diff
@@ -233,6 +234,7 @@ public void isRegisteredExceptionMapperNullExceptionTest() throws Fault {
    * none
    */
   @Test
+  @Disabled("In response to accepted Platform TCK challenge https://github.com/eclipse-ee4j/jaxrs-api/issues/937")
```

Comment:
This does not need to be disabled since [a patch](https://github.com/eclipse-ee4j/jakartaee-tck/pull/588) is already in.

Reply:
Enabled now.